app/controllers/concerns/audits1984.rb

```diff
@@ -0,0 +1,7 @@
+module Audits1984
+  private
+
+  def find_current_auditor
+    Current.user if Current.user.admin?
```

Comment:
I'm not sure if this suggestion is necessary. I'm thinking `Current.user` probably can't be `nil` since the route concern prevents non-admins from accessing. I'm leaving this one up to you

```suggestion
    Current.user if Current.user&.admin?
```

Reply:
I think it's a good idea to implement this, but just to clarify the reason I added it here as well as under the constraint is to make it clear it's just for admins - I assume the constraint wouldn't be removed by accident so it's not really a technical safeguard.